eng/common/tools.sh

```diff
@@ -455,6 +455,7 @@ function MSBuild-Core {
     }
   }
 
+  echo "$_InitializeBuildToolCommand /m /nologo /clp:Summary /v:$verbosity /nr:$node_reuse $warnaserror_switch /p:TreatWarningsAsErrors=$warn_as_error /p:ContinuousIntegrationBuild=$ci $@"
```

Comment:
Same comment about Arcade owning this.  The next Arcade update will overwrite this.

Reply:
ditto